### ISSUE TITLE
Connect to GitHub and retrieve the issues list in a CSV

### ISSUE THREAD

**Issue by javierobledo** [JSwingRipples2016/jswingripples#72]

**Comment by druzmieres** [comment]
We have to check if each XML tag can have associated values inside.

**Comment by druzmieres** [comment]
Instead of saving the colors as hexadecimal, there should be tags in the beginning of the XML files, these tags should be associated with a color. Instead of writing <#32312> ... <\#32312>, we should write <blue> ... <\blue>.

**Comment by jestebanlh** [comment]
Yeah according to class discussion the format in which the issues will be saved is a XML. For this reason the issue name have been changed.

**Comment by cfierro94** [comment]
<script src="https://gist.github.com/cfierro94/a3110f8a57f03c6e76c2cd90f11b26c9.js"></script>

**Comment by cfierro94** [comment]
https://gist.github.com/cfierro94/a3110f8a57f03c6e76c2cd90f11b26c9

**Comment by cfierro94** [comment]
<?xml version="1.0" encoding="UTF-8"?>
<project>
    <name>Dummy</name>
    <list_issues>
        <issue id="5672">
            <number>1</number>
            <title>Soy una issue</title>
            <commments>"Crear un <color><param>red</param>botón</color> hola"</commments>
        </issue>
    </list_issues>
</project>

**Comment by jestebanlh** [comment]
Source: Team / Teacher
Priority: High
Stability: Non-Negotiable
Last Update: 2016-11-02
State: Fulfils
Increment: 1
Type: Interation
Associated user: End-User

User requirements associated:

    RU009: User Interface Enhancement

Software Change Process:

    Concept Location
    Impact Analysis
    Prefactoring
    Actualization
    Postfactoring
    Conclusion

**Comment by jestebanlh** [comment]
Classes vinculated with the class JavaProject

![imagen](https://cloud.githubusercontent.com/assets/22397881/21205918/8ad1bd30-c23d-11e6-88ec-dace5850c186.png)

![imagen](https://cloud.githubusercontent.com/assets/22397881/21205971/dc6fc1f0-c23d-11e6-91b5-33db2d17a243.png)
